### PR TITLE
[doc] Adjust private registry support statement

### PR DESCRIPTION
**Pull request by sebsoto**
Removes statement on lack of support for pulling images from a registry namespace which requires authentication. It is possible to pull images by authenticating with a kubernetes.io/dockerconfigjson secret.

### CODE REVIEW

**Comment by mansikulkarni96** [comment]
While removing this section, should we add a doc pointing to the official docs on authenticating with a kubernetes.io/dockerconfig.json secret?

**Comment by sebsoto** [comment]
https://github.com/openshift/windows-machine-config-operator/pull/2168#issuecomment-2107810780

No, I dont think so. For the same reason we don't link to docs on how to create deployments. It's general kubernetes functionality, not specific to openshift.

File: README.md [hunk 1]
```diff
@@ -252,10 +252,6 @@ WMCO / Windows nodes does not work with the following products:
 * [OpenShift monitoring of user defined project](https://docs.openshift.com/container-platform/latest/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects)
 * [HugePages](https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/)
 
-### Accessing secure registries
```
Comment (saifshaikh48) [inline README.md:255]: should we list private registry access in the "Enabled features" section?

**Review by mansikulkarni96 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jrvaldes) [reply README.md:255]: have mixed feelings about that, if so I would  join Mansi's statement  https://github.com/openshift/windows-machine-config-operator/pull/2168#issuecomment-2107810780

**Comment by jrvaldes** [comment]
@sebsoto  Please mention in the commit message "why" this statement is no longer relevant and "what" changed which invalidates  https://github.com/openshift/windows-machine-config-operator/pull/1174

**Comment by jrvaldes** [comment]
@sebsoto  @mansikulkarni96  please override the e2e test jobs for this docs PR

**Comment by sebsoto** [comment]
@jrvaldes Nothing has changed, I have given all the information.

**Comment by sebsoto** [comment]
/overrride ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/wicd-unit-vsphere ci/prow/ci-bundle-wmco-bundle

**Comment by sebsoto** [comment]
/overrride ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/wicd-unit-vsphere ci/prow/ci-bundle-wmco-bundle

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/wicd-unit-vsphere ci/prow/ci-bundle-wmco-bundle

**Pull request merged**